modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * osd_cluster_create/managing-byo-vpc-multiple-az.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= Configuring subnets for a VPC with multiple availability zones
+
+For VPCs with multiple availability zones, you must configure a `public` and `private` subnet per availability zone, and then, create the appropriate routing rule and NAT gateway for each of the availability zones.
+
+.Procedure
+
+. Create six subnets for three separate zones as follows:
++
+[cols="3",options="header"]
+|===
+| Availability zone | Subnet name | CIDR IP address
+| A | `publicA`  | `10.0.0.0/19`
+| A | `privateA` | `10.0.96.0/19`
+| B | `publicB`  | `10.0.32.0/19`
+| B | `privateB` | `10.0.128.0/19`
+| C | `publicC`  | `10.0.64.0/19`
+| C | `privateC` | `10.0.160.0/19`
+|===
++
+[NOTE]
+====
+Each subnet is for a separate zone, each with a `public` and `private` subnet.
+====
+
+. Create a `RouteTable` that is associated with the three `public` subnets.
```

Comment:
`associated` is a general word and may cause confusion as its actually a component of AWS route tables. We should link to AWS documentation on how to associated subnets with route tables? https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html#RouteTables